components/script/dom/webglframebuffer.rs

```diff
@@ -251,7 +251,8 @@ impl WebGLFramebuffer {
         Ok(())
     }
 
-    pub fn detach_renderbuffer(&self, rb: &WebGLRenderbuffer) {
+    fn call_on_matching_renderbuffers<F>(&self, rb: &WebGLRenderbuffer, closure: F)
+        where F: Fn(&DOMRefCell<Option<WebGLFramebufferAttachment>>) {
```

Comment:
nit: naming convention for this kind of function is `with`, like `with_matching_{renderbuffers/textures}`.

Also, when there's a `where` clause usually the opening brace goes on the next line aligned with the `fn`:

```rust
fn with_matching_renderbuffers<F>(&self, rb: &WebGLRenderbuffer, closure: F)
    where F: Fn(&DOMRefCell<Option<WebGLFramebufferAttachment>>)
{
    // ...
```